examples/user_guide/Deploy_and_Export.ipynb

```diff
@@ -549,6 +551,53 @@
     "This way we can create a global indicator for the busy state instead of modifying all our callbacks."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### pn.state.schedule\n",
+    "\n",
+    "The `pn.state.schedule` functionality allows scheduling global tasks which will be run in a global context at a periodic interval. Note that the scheduled task **may not** be declared in the application code itself, i.e. if you are serving `panel serve app.py` the callback you are scheduling must not be declared in the `app.py`. It must be defined in an external module or in a `--setup` script declared as part of the `panel serve` invocation.\n",
+    "\n",
+    "Scheduling using `pn.state.schedule` is idempotent, i.e. if a callback has already been scheduled under the same name subsequent calls will have no effect. By default the starting time is immediate but may be overridden with the `at` keyword argument. The period may be declared using the `period` argument or a cron expression (which requires the `croniter` library). Note that the `at` time should be in local time but if a callable is provided it must return a UTC time. If `croniter` is installed a `cron` expression can be provided using the `cron` argument.\n",
```

Comment:
Shouldn't subsequent calls with the same name but a different callback remove the previous callback and replace it? I think that's what people would expect; I'd be very surprised if I called it twice in a row but got the first callback rather than the second.

Reply:
The problem with this is that if you put the `pn.state.schedule` statement inside your app (which is valid as long as you import the callback from some other module) then it will re-schedule the same task over an over again and therefore mess up the scheduling if you used a relative rather than an absolute period.

Reply:
Ah. Sounds like something to explain in the docs so that other people don't have my reaction.